opennms-doc/guide-install/src/asciidoc/text/newts/cassandra-windows.adoc

```diff
@@ -0,0 +1,26 @@
+
+// Allow GitHub image rendering
+:imagesdir: ../../images
+
+[[gi-install-cassandra-windows]]
+==== Installing on Windows Server systems
+
+This section describes how to install the latest _Cassandra 2.1.x_ release on a _Microsoft Windows Server_ based systems for _Newts_.
+The first steps download the graphical installer and register _Cassandra_ as a _Windows Service_ and can be manged through the _Service Manager_.
+
+NOTE: This description is build on _Windows Server 2012_.
+
+.Download the DataStax graphical installer for Cassandra from PowerShell or a Browser
+[source]
+----
+cd C:\Users\Administrator\Downloads
+Invoke-WebRequest http://downloads.datastax.com/community/datastax-community-64bit_2.1.8.msi -Outfile datastax-community-64bit_2.1.8.msi
+----
+
+Run the Windows Installer file from _PowerShell_ or through _Windows Explorer_ and follow the setup wizard to install.
+During the installation, accept the options to automatically start the services.
+By default the _DataStax Server_, _OpsCenter Server_ and the _OpsCenter Agent_ will be automatically installed and started.
+
+NOTE: The _DataStax OpsCenter Server_ is only required to be installed once per _Cassandra Cluster_.
+
+IMPORTANT: If you install the _DataStax OpsCenter_ make sure you have _Chrome_ or _Firefox_ as browser installed.
```

Comment:
If you install the _DataStax OpsCenter_ make sure you have _Chrome_ or _Firefox_ installed.